### PR TITLE
Aws docs 2a

### PR DESCRIPTION
**Pull request by ghost**
commits related to aws documents

### CODE REVIEW

File: html5/_main_toc.html.slim [hunk 1]
```diff
@@ -189,6 +189,9 @@ nav#toc.collapse.bd-links.toc
           a href="https://docs.killbill.io/latest/Kill-Bill-Glossary.html"
             | Glossary
   ul.nav.navbar-nav
+        li
+          a href="https://docs.killbill.io/latest/Kill-Bill-Glossary.html"
+            | Glossary
     li
```
Comment (reshmabidikar) [inline html5/_main_toc.html.slim:191]: There is an extra Glossary link. This messes up the build, I had deleted it earlier but looks like you have an older version of this file. Can you please delete it?

File: userguide/tutorials/aws-container.adoc [hunk 2]
```diff
@@ -59,15 +69,17 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
 [start=6]
 . Choose FARGATE as the launch type and select an existing VPC and subnet:
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-run-task-config.png[align=center]
+image:ecs-run-task.png[align=center]
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:73]: Should'nt the image name be `ecs-run-task-config.png`?

File: html5/_main_toc.html.slim (hunk 1 above)
Comment (ghost) [reply html5/_main_toc.html.slim:191]: @reshmabidikar - Fixed...please check

File: userguide/tutorials/aws-container.adoc (hunk 2 above)
Comment (ghost) [reply userguide/tutorials/aws-container.adoc:73]: @reshmabidikar - Fixed...please check

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**